client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -40,21 +40,33 @@ class SiteTools extends Component {
 		const themeSetupLink = `/settings/theme-setup/${ selectedSite.slug }`;
 		const startOverLink = `/settings/start-over/${ selectedSite.slug }`;
 		const deleteSiteLink = `/settings/delete-site/${ selectedSite.slug }`;
-		let changeAddressLinkText = translate( 'Register a new domain or change your site\'s address.' );
-		const themeSetupLinkText = translate( 'Automatically make your site look like your theme\'s demo.' );
-		const strings = {
-			changeSiteAddress: translate( 'Change your site address' ),
-			themeSetup: translate( 'Theme setup' ),
-			startOver: translate( 'Delete your content' ),
-			deleteSite: translate( 'Delete your site permanently' )
-		};
 
-		if ( ! this.props.hasLoadedSitePurchasesFromServer ) {
-			return null;
-		}
+		const themeSetupText = translate( 'Automatically make your site look like your theme\'s demo.' );
+		const changeSiteAddress = translate( 'Change your site address' );
+		const themeSetup = translate( 'Theme setup' );
+		const startOver = translate( 'Delete your content' );
+		const startOverText = translate(
+			'Keep your site\'s address and current theme, but remove all posts, ' +
+			'pages, and media so you can start fresh.'
+		);
+		const deleteSite = translate( 'Delete your site permanently' );
+		const deleteSiteText = translate(
+			'Delete all your posts, pages, media and data, ' +
+			'and give up your site\'s address'
+		);
+
+		const importTitle = translate( 'Import' );
+		const importText = translate( 'Import content from another WordPress or Medium site' );
+		const exportTitle = translate( 'Export' );
+		const exportText = translate( 'Export content from your site. You own your data' );
 
+		let changeAddressText = translate( 'Register a new domain or change your site\'s address.' );
 		if ( ! config.isEnabled( 'upgrades/domain-search' ) ) {
-			changeAddressLinkText = translate( 'Change your site address.' );
+			changeAddressText = translate( 'Change your site address.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **39** times:
`translate( 'Change Site Address' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).